javascript/javascript1/week3/index2.js

```diff
@@ -0,0 +1,71 @@
+
+let notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  
+    let obj={
+        content:content,
+        id:id
+    }
+    
+    notes.push(obj)
+}
+  
+
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes);
+
+function getNote(id) {
+    // your code here
+    for (i=0;i <=notes.length;i++){
+        if(id != ""){
```

Comment:
id type is the number, and you check here if id is not a empty string, it always true.